resources/js/layouts/marketing-layout.tsx

```
@@ -0,0 +1,56 @@
+import AppLogoIcon from '@/components/app-logo-icon';
+import { Button } from '@/components/ui/button';
+import { login, register } from '@/routes/index';
```

ESLint

[error] 'register' is defined but never used.

(`@typescript-eslint/no-unused-vars`)

---

[error] `react` type import should occur before import of `@/components/app-logo-icon`

(import/order)